src/components/MainTable/MainTable.tsx

```diff
@@ -1,15 +1,58 @@
 import PropTypes from "prop-types";
 import React, { useEffect, useState } from "react";
+import type { ReactNode } from "react";
+import type { SortDirection } from "types";
 
 import Pagination from "../Pagination";
 import Table from "../Table";
+import type { TableProps } from "../Table";
 import TableRow from "../TableRow";
 import TableHeader from "../TableHeader";
 import TableCell from "../TableCell";
 
 import "./MainTable.scss";
+import { TSFixMe } from "index";
 
-const updateSort = (setSortKey, setSortDirection, sortKey, sortDirection) => {
+type Header = {
+  content?: ReactNode;
+  className?: string;
+  sortKey?: string;
+};
+
+type Column = {
```

Comment:
Could this type be imported from the TableCell component?

Reply:
Are TableCells and Columns equivalent?

Reply:
The Column props get spread onto a TableCell so the props/types should be the same. You're probably right, these should use the same name.